homeassistant/components/braviatv/media_player.py

```diff
@@ -294,11 +303,15 @@ def set_volume_level(self, volume):
 
     def turn_on(self):
         """Turn the media player on."""
+        self._state_change = True
```

Comment:
Why do we need this?

If we need to guard from parallel updates, set `PARALLEL_UPDATES` constant to 1 here in the media_player module.

Reply:
I only wanted to protect against updating during weird states. Is ```PARALLEL_UPDATES```  a semaphore per TV or all TVs under this integration? ```turn_on()``` has a timeout of 10 seconds which it will commonly get close to if the TV is coming from a completely off state -- I wouldn't want to stop updates to other TVs because another TV is being turned on.

Reply:
The `PARALLEL_UPDATES` semaphore is for all entities of the platform, ie all TVs for this integration.

If we want to guard the device interface just within this entity I think we would need an asyncio.Lock. It's not enough with just this boolean attribute `_state_change`, since the `turn_on` method will be run in the thread pool while `async_update` is run in the event loop, ie different threads.

Reply:
Done. I hope it was acceptable that I choose not to attempt to acquire the lock during async_update(). Since bravia.turn_on() can take a bit, it would push async_update() into it's next update interval.